templates/zerver/help/create-a-stream.md

```diff
@@ -0,0 +1,40 @@
+# Create a stream
+On Zulip, streams are similar to conversation threads or subjects. Streams are located on the left sidebar.
+Streams can be either:
+* **Public streams** - Public streams are for open discussions. All users can subscribe to public streams and discuss there.
+* **Private streams** - Private streams are for confidential discussions and are only visible to users who've been invited to subscribe to it.
```

Comment:
Nitpick: `subscribe to it` should be `subscribe to them` to agree with the plural `Private streams`